naucse/models.py

```diff
@@ -269,6 +273,8 @@ def materials(self):
 
         return materials
 
+    def github_link(self, run, coverpage):
+        return "https://github.com/pyvec/naucse.python.cz/blob/master/runs/" + run + "/sessions/" + self.slug + "/" + coverpage + ".md"
```

Comment:
Please add a function to the model root that generates this link from a Path.
Github is only our current hosting provider; their URL scheme should be hardcoded in one place at most.